scripts/build/companion_libs/350-newlib_nano.sh

```diff
@@ -205,12 +231,13 @@ ENABLE_TARGET_OPTSPACE:target-optspace
 -isystem %:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/include) %(newlib_nano_cpp)
 
 *cc1plus:
--idirafter %:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/include) %(newlib_nano_cc1plus)
+-idirafter %:getenv(GCC_EXEC_PREFIX ../../${CT_TARGET}/include/newlib-nano) %(newlib_nano_cc1plus)
 
-*link:
--L%:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/lib/%M) -L%:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/lib)
 
+*link:
+%(nano_link) -L%:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/lib/%M) -L%:getenv(GCC_EXEC_PREFIX ../../newlib-nano/${CT_TARGET}/lib)
```

Comment:
Since we are now using the original `nano.specs` for `CT_NEWLIB_NANO_INSTALL_IN_TARGET=y`, which does indeed support `_nano` suffixed libraries in the main library directory, these should be reverted.

The previously suggested changes were for using Keith's `nano.specs` with `CT_NEWLIB_NANO_INSTALL_IN_TARGET=y`. When `CT_NEWLIB_NANO_INSTALL_IN_TARGET=n`, the original implementation is correct and need not be changed.